bye-bye-jetifier/src/main/java/com/dipien/byebyejetifier/ProjectAnalyzer.kt

```diff
@@ -132,7 +133,7 @@ class ProjectAnalyzer(
     private fun filterSupportAnnotationsIfNeeded(results: List<ScanResult>): List<ScanResult> {
         if (excludeSupportAnnotations) {
             return results.filter { scanResult ->
-                !scanResult.legacyDependency.startsWith("android/support/annotation") &&
+                !scanResult.legacyDependency.startsWith("android/support/annotation".toFilePath()) &&
```

Comment:
Maybe you could move the string to a constant to avoid transforming it multiple times